File: Admin.NET/Admin.NET.Application/Entity/WmsMoveNotify.cs
// Admin.NET 项目的版权、商标、专利和其他相关权利均受相应法律法规的保护。使用本项目应遵守相关法律法规和许可证的要求。
//
// 本项目主要遵循 MIT 许可证和 Apache 许可证（版本 2.0）进行分发和使用。许可证位于源代码树根目录中的 LICENSE-MIT 和 LICENSE-APACHE 文件。
//
// 不得利用本项目从事危害国家安全、扰乱社会秩序、侵犯他人合法权益等法律法规禁止的活动！任何基于本项目二次开发而产生的一切法律纠纷和责任，我们不承担任何责任！

using Admin.NET.Core;
using SqlSugar;
namespace Admin.NET.Application.Entity;

/// <summary>
/// 移库单据明细表
/// </summary>
[Tenant("1300000000001")]
[SugarTable("WmsMoveNotify", "移库单据明细表")]
public partial class WmsMoveNotify : EntityBaseDel
{
    /// <summary>
    /// 移库单据类型 -- 单号
    /// </summary>
    [SugarColumn(ColumnName = "MoveBillCode", ColumnDescription = "移库单据类型", Length = 50)]
    public virtual string? MoveBillCode { get; set; }
    
    /// <summary>
    /// 移库序号
    /// </summary>
    [SugarColumn(ColumnName = "MoveListNo", ColumnDescription = "移库序号")]
    public virtual int? MoveListNo { get; set; }
    
    /// <summary>
    /// 移库批次
    /// </summary>
    [SugarColumn(ColumnName = "MoveLotNo", ColumnDescription = "移库批次", Length = 50)]
    public virtual string? MoveLotNo { get; set; }
    
    /// <summary>
    /// 物料id
    /// </summary>
    [SugarColumn(ColumnName = "MaterialId", ColumnDescription = "物料id", Length = 50)]
    public virtual string? MaterialId { get; set; }
    
    /// <summary>
    /// 移库物料型号
    /// </summary>
    [SugarColumn(ColumnName = "MoveMaterialModel", ColumnDescription = "移库物料型号", Length = 100)]
    public virtual string? MoveMaterialModel { get; set; }
    
    /// <summary>
    /// 物料温度
    /// </summary>
    [SugarColumn(ColumnName = "MoveMaterialTemp", ColumnDescription = "物料温度", Length = 50)]
    public virtual string? MoveMaterialTemp { get; set; }
    
    /// <summary>
    /// 物料类型（字典MaterialType）
    /// </summary>
    [SugarColumn(ColumnName = "MoveMaterialType", ColumnDescription = "物料类型（字典MaterialType）", Length = 100)]
    public virtual string? MoveMaterialType { get; set; }
    
    /// <summary>
    /// 物料状态
    /// </summary>
    [SugarColumn(ColumnName = "MoveMaterialStatus", ColumnDescription = "物料状态", Length = 100)]
    public virtual string? MoveMaterialStatus { get; set; }
    
    /// <summary>
    /// 物料单位
    /// </summary>
    [SugarColumn(ColumnName = "MoveMaterialUnit", ColumnDescription = "物料单位", Length = 50)]
    public virtual string? MoveMaterialUnit { get; set; }
    
    /// <summary>
    /// 物料品牌
    /// </summary>
    [SugarColumn(ColumnName = "MoveMaterialBrand", ColumnDescription = "物料品牌", Length = 100)]
    public virtual string? MoveMaterialBrand { get; set; }
    
    /// <summary>
    /// 生产日期
    /// </summary>
    [SugarColumn(ColumnName = "MoveProductionDate", ColumnDescription = "生产日期")]
    public virtual DateTime? MoveProductionDate { get; set; }
    
    /// <summary>
    /// 失效日期
    /// </summary>
    [SugarColumn(ColumnName = "MoveLostDate", ColumnDescription = "失效日期")]
    public virtual DateTime? MoveLostDate { get; set; }
    
    /// <summary>
    /// 移库数量
    /// </summary>
    [SugarColumn(ColumnName = "MoveQuantity", ColumnDescription = "移库数量")]
    public virtual int? MoveQuantity { get; set; }
    
    /// <summary>
    /// 移库日期
    /// </summary>
    [SugarColumn(ColumnName = "MoveDate", ColumnDescription = "移库日期")]
    public virtual DateTime? MoveDate { get; set; }
    
    /// <summary>
    /// 仓库id
    /// </summary>
    [SugarColumn(ColumnName = "MoveWarehouseId", ColumnDescription = "仓库id")]
    public virtual long? MoveWarehouseId { get; set; }
    
    /// <summary>
    /// 移出巷道id
    /// </summary>
    [SugarColumn(ColumnName = "MoveLanewayOutCode", ColumnDescription = "移出巷道id")]
    public virtual long? MoveLanewayOutCode { get; set; }
    
    /// <summary>
    /// 移入巷道id
    /// </summary>
    [SugarColumn(ColumnName = "MoveLanewayInCode", ColumnDescription = "移入巷道id", Length = 50)]
    public virtual string? MoveLanewayInCode { get; set; }
    
    /// <summary>
    /// 移出储位编码
    /// </summary>
    [SugarColumn(ColumnName = "MoveOutSlotCode", ColumnDescription = "移出储位编码", Length = 50)]
    public virtual string? MoveOutSlotCode { get; set; }
    
    /// <summary>
    /// 移入储位编码
    /// </summary>
    [SugarColumn(ColumnName = "MoveInSlotCode", ColumnDescription = "移入储位编码", Length = 50)]
    public virtual string? MoveInSlotCode { get; set; }
    
    /// <summary>
    /// 执行标志（01待执行、02正在执行、03已完成、04已上传）
    /// </summary>
    [SugarColumn(ColumnName = "MoveExecuteFlag", ColumnDescription = "执行标志（01待执行、02正在执行、03已完成、04已上传）", Length = 50)]
    public virtual string? MoveExecuteFlag { get; set; }
    
    /// <summary>
    /// 备注
    /// </summary>
    [SugarColumn(ColumnName = "MoveRemark", ColumnDescription = "备注", Length = 100)]
    public virtual string? MoveRemark { get; set; }
    
    /// <summary>
    /// 移库任务号
    /// </summary>
    [SugarColumn(ColumnName = "MoveTaskNo", ColumnDescription = "移库任务号", Length = 100)]
    public virtual string? MoveTaskNo { get; set; }
    
    /// <summary>
    /// 库存表id
    /// </summary>
    [SugarColumn(ColumnName = "StockStockCodeId", ColumnDescription = "库存表id")]
    public virtual string? StockStockCodeId { get; set; }
    
    /// <summary>
    /// 软删除
    /// </summary>
    [Required]
    [SugarColumn(ColumnName = "IsDelete", ColumnDescription = "软删除")]
    public virtual bool IsDelete { get; set; }
    
}

File: Web/src/api/move/moveNotify/wmsMoveNotify.ts
import {useBaseApi} from '/@/api/base';

// 移库单据明细表接口服务
export const useWmsMoveNotifyApi = () => {
	const baseApi = useBaseApi("wmsMoveNotify");
	return {
		// 分页查询移库单据明细表
		page: baseApi.page,
		// 查看移库单据明细表详细
		detail: baseApi.detail,
		// 新增移库单据明细表
		add: baseApi.add,
		// 更新移库单据明细表
		update: baseApi.update,
		// 删除移库单据明细表
		delete: baseApi.delete,
		// 批量删除移库单据明细表
		batchDelete: baseApi.batchDelete,
		// 导出移库单据明细表数据
		exportData: baseApi.exportData,
		// 导入移库单据明细表数据
		importData: baseApi.importData,
		// 下载移库单据明细表数据导入模板
		downloadTemplate: baseApi.downloadTemplate,
	}
}

// 移库单据明细表实体
export interface WmsMoveNotify {
	// 主键Id
	id: number;
	// 移库单据类型
	moveBillCode: string;
	// 移库序号
	moveListNo: number;
	// 移库批次
	moveLotNo: string;
	// 物料id
	materialId: string;
	// 移库物料型号
	moveMaterialModel: string;
	// 物料温度
	moveMaterialTemp: string;
	// 物料类型（字典MaterialType）
	moveMaterialType: string;
	// 物料状态
	moveMaterialStatus: string;
	// 物料单位
	moveMaterialUnit: string;
	// 物料品牌
	moveMaterialBrand: string;
	// 生产日期
	moveProductionDate: string;
	// 失效日期
	moveLostDate: string;
	// 移库数量
	moveQuantity: number;
	// 移库日期
	moveDate: string;
	// 仓库id
	moveWarehouseId: number;
	// 移出巷道id
	moveLanewayOutCode: number;
	// 移入巷道id
	moveLanewayInCode: string;
	// 移出储位编码
	moveOutSlotCode: string;
	// 移入储位编码
	moveInSlotCode: string;
	// 执行标志（01待执行、02正在执行、03已完成、04已上传）
	moveExecuteFlag: string;
	// 备注
	moveRemark: string;
	// 移库任务号
	moveTaskNo: string;
	// 库存表id
	stockStockCodeId: number;
	// 创建者Id
	createUserId: number;
	// 创建时间
	createTime: string;
	// 修改者Id
	updateUserId: number;
	// 更新时间
	updateTime: string;
	// 软删除
	isDelete?: boolean;
	// 创建者姓名
	createUserName: string;
	// 修改者姓名
	updateUserName: string;
}